src/backend/src/routes/design-reviews.routes.ts

```diff
@@ -38,7 +38,7 @@ designReviewsRouter.post(
   body('isInPerson').isBoolean(),
   nonEmptyString(body('zoomLink')).isURL().optional(),
   nonEmptyString(body('location')).optional(),
-  nonEmptyString(body('docTemplateLink')).isURL().optional(),
```

Comment:
Why remove this

Reply:
i was submitting a url and it wasnt passing